src/main/java/com/metamx/emitter/core/HttpEmitterConfig.java

```diff
@@ -134,6 +138,31 @@ public HttpEmitterConfig(
       long maxBufferSize
   )
   {
+    this(
```

Comment:
We now have 6 overriding constructors, do you think it would be practical to introduce Builder and deprecate constructors?